pmb/helpers/other.py

```
@@ -78,29 +78,30 @@ def check_binfmt_misc(args):
                        " armhf on x86_64):\n See: <" + link + ">")
 
 
-def migrate_success(args):
-    logging.info("Migration done")
+def migrate_success(args, version):
+    logging.info("Migration to version " + str(version) + " done")
     with open(args.work + "/version", "w") as handle:
-        handle.write(pmb.config.work_version + "\n")
+        handle.write(str(version) + "\n")
 
 
 def migrate_work_folder(args):
     # Read current version
-    current = "0"
+    current = 0
     path = args.work + "/version"
     if os.path.exists(path):
         with open(path, "r") as f:
-            current = f.read().rstrip()
+            current = int(f.read().rstrip())
 
     # Compare version, print warning or do nothing
     required = pmb.config.work_version
     if current == required:
```

@ollieparanoid what you think about adding a check if the current version is greater than the required?
Do the versions are backward compatibles? otherwise it's better to raise an error

---

You're right, it's better to have such a check!